styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,77 @@
+a.footer-menu-link {
+  color: darken($light-gray, 15%);
+  font-size: 1.2rem;
+}
+a.footer-tab-link{
+  color:$white;
+  font-size: 1.3rem;
+}
 .footer{
-  
-}
\ No newline at end of file
+  ul{
```

Comment:
I'd rename this `.footer-tabs` and then just remove list style, padding, and margin from `.footer-menu` separately.